packages/cli/src/constants.ts

```diff
@@ -41,3 +41,9 @@ export const UNKNOWN_FAILURE_REASON = 'Unknown failure reason';
 
 export const WORKFLOW_REACTIVATE_INITIAL_TIMEOUT = 1000;
 export const WORKFLOW_REACTIVATE_MAX_TIMEOUT = 180000;
+
+export const SETTINGS_LICENSE_CERT_KEY = 'license.cert';
+
+export enum LICENSE_FEATURES {
```

Comment:
Instead of using an `enum`, should we follow the same pattern as above, like:

```typescript
export const LICENSE_FEATURES = {
	SHARING: 'feat:sharing'
};
```

Reply:
Also, I believe Product suggested to make workflow- and credential sharing two separate feature toggles.